packages/apps/spaces/ui/form/DatePicker/DatePickerUnderline.tsx

```diff
@@ -92,6 +90,7 @@ export const DatePickerUnderline: React.FC<DatePickerProps> = ({
             DateTimeUtils.abreviatedMonth,
           )
         }
+        value={value as Date}
```

Comment:
I see that this one is done with Chakra. I know that Mihai and Steyn worked on a new aligned DatePicker or Calendar in `DatePicker2.tsx` which is done with tw. Would be worth migrating to that one as we'll pretty soon remove chakra from the codebase.

Reply:
it's not chakra, it's react-date-picker. I will remove the Box as the last part of chakra remaining

Reply:
ok but we still plan on removing every other DatePicker and using the one just implemented.

Reply:
Ok, let's get this through and circle back later when we approach removing chakra.